Week2/homework/first-homework.js

```diff
@@ -0,0 +1,131 @@
+const mysql = require('mysql');
+const util = require('util');
+const program = require('commander');
+
+const connection_config = mysql.createConnection({
+  host: 'localhost',
+  user: 'hyfuser',
+  password: 'hyfpassword',
+  database: 'new_world',
+});
+
+const execQuery = util.promisify(connection_config.query.bind(connection_config));
+
+connection_config.connect(err => {
+  if (err) {
+    throw err;
+  }
+});
+
+program.version('0.0.1');
+
+program.description(
+  'Hello! Welcome to this small cli program.\nHere we provide you some information based on what you specify.\nLook below and start exploring!',
+);
+
+program
+  .command('capital-of <country>')
+  .description('This command gives you the capital of the country')
+  .action(country => {
+    const getCapital = `SELECT city.name as Capital 
+    FROM country, city 
+    WHERE country.capital = city.id 
+    AND country.name ='${country}';`;

```

Comment:
Be careful, this code in vulnerable to attack, someone could easily drop your data. That is why the homework asked to use "prepared query" using the annotation <?> .
You can have a look in async-create-insert.js how the student is inserted.
Otherwise the query is good